frontend/src/layouts/Footer.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useTenant } from 'tenants';
+
+/**
+ * Footer component that applies tenant environmental styling.
+ * @param param0 Footer properties.
+ * @returns Footer component.
+ */
+export const Footer: React.FC<React.HTMLAttributes<HTMLElement>> = ({ ...rest }) => {
+  const tenant = useTenant();
+  return <FooterStyled {...rest} backgroundColor={tenant.colour}></FooterStyled>;
+};
+
+interface IFooterProps {
+  // Background color of footer.
+  backgroundColor: string;
+}
+
+const FooterStyled = styled.footer<IFooterProps>`
+  background-color: ${props => props.backgroundColor};
+
+  // colors
+  background-color: $primary-color;
```

Comment:
hmm, does this syntax work? I thought we needed ${variables.primaryColor} in these blocks.

Reply:
No it doesn't.  I have a separate PR I'm working on that will apply these variables to styled component themes.